src/neptune/cli/sync.py

```diff
@@ -50,60 +51,122 @@
 
 class SyncRunner:
     @staticmethod
-    def sync_all_offline(*, backend: "NeptuneBackend", base_path: Path, project_name: Optional[str] = None) -> None:
+    def sync_all_offline(
+        *,
+        backend: "NeptuneBackend",
+        base_path: Path,
+        project_name: Optional[str] = None,
+        num_threads: Optional[int] = None,
+    ) -> None:
         containers = collect_containers(path=base_path, backend=backend)
 
-        project = get_project(project_name_flag=QualifiedName(project_name) if project_name else None, backend=backend)
+        project = get_project(
+            project_name_flag=QualifiedName(project_name) if project_name else None,
+            backend=backend,
+        )
```

Comment:
I assume this change is related to automatic formatting, right?

Reply:
yup